### PR TITLE
Fuzzy logic added for Headers suggestions

### PR DESCRIPTION
**Pull request by BrawlerXull**
## PR Description

Fuzzy logic added for Headers suggestions

## Related Issues
#207 
- Related Issue #
- Closes #

### Checklist
- [x] I have gone through the [contributing guide](https://github.com/foss42/apidash/blob/main/CONTRIBUTING.md)
- [x] I have run the tests (`flutter test`) and all tests are passing

## Added/updated tests?
_We encourage you to add relevant test cases._

- [x] Yes
- [ ] No, and this is why: _please replace this line with details on why tests have not been included_
- [ ] I need help with writing tests

### CODE REVIEW

**Comment by animator** [comment]
Instead of modifying the existing `getHeaderSuggestions` add a new function `getFuzzyHeaderSuggestions` and plug it where `getHeaderSuggestions` was being used so that we can switch it if there is any issue.
Also do not modify the existing test cases. Add new test cases for `getFuzzyHeaderSuggestions`.

File: lib/utils/header_utils.dart [hunk 1]
```diff
@@ -105,4 +113,4 @@ List<String> getHeaderSuggestions(String pattern) {
   final suggestions = results.map((result) => result.item).toList();
 
   return suggestions;
-}
+}
```
Comment (animator) [inline lib/utils/header_utils.dart:116]: Revert deleting this newline from the End of the file.

**Comment by animator** [comment]
@BrawlerXull any updates?

**Comment by ashitaprasad** [comment]
No problem @BrawlerXull

**Comment by ashitaprasad** [comment]
Please open a new PR which provides improved results as discussed above.

**Pull request closed without merging**